k8s-jobs.yml

```diff
@@ -24,7 +26,18 @@ jobs:
   - {name: clamav, release: clamav}
   - {name: snort, release: snort}
 - name: etcd
-  templates: *hardening-templates
+  templates:
```

Comment:
I guess we could redeclare this stanza later in the file with another `(( append ))`, but that seems worse.